<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeStatisticsClass</class>
 <widget class="QWidget" name="TimeStatisticsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>227</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TimeStatistics</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background-color:rgb(60,60,60);
	border-radius:12;
	border:2px solid #4F4F4F;
}

QFrame#frame{
	background-color:rgb(50,50,50);
	border:none;
	border-radius:0;
}

QLabel{
	background: &quot;transparent&quot;;
	border:none;
    color:white
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7" rowstretch="1,6">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="MyLogoButton" name="btn_ControlVisiable" native="true">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ReSet">
          <property name="minimumSize">
           <size>
            <width>62</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>62</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
	color:#999999;
}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Resource/BtnReSet.png</normaloff>Resource/BtnReSet.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_ControlVisual" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="1">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:10pt;
color:#999999;</string>
                </property>
                <property name="text">
                 <string>Average</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="0" rowspan="2">
               <widget class="QLabel" name="label_Average">
                <property name="minimumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image:url(&quot;:/DynamicWidget/Resource/TimeStatisticAverage.png&quot;);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_AverageTime">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>13</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:13pt;</string>
                </property>
                <property name="text">
                 <string> 0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0" rowspan="2">
               <widget class="QLabel" name="label_Max">
                <property name="minimumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image:url(&quot;:/DynamicWidget/Resource/TimeStatisticMax.png&quot;);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_MaxTime">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>13</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:13pt;</string>
                </property>
                <property name="text">
                 <string> 0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:10pt;
color:#999999;</string>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0" rowspan="2">
               <widget class="QLabel" name="label_Min">
                <property name="minimumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image:url(&quot;:/DynamicWidget/Resource/TimeStatisticMin.png&quot;);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_MinTime">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>13</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:13pt;</string>
                </property>
                <property name="text">
                 <string> 0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:10pt;
color:#999999;</string>
                </property>
                <property name="text">
                 <string>Min</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0" rowspan="2">
               <widget class="QLabel" name="label_OverTime">
                <property name="minimumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image:url(&quot;:/DynamicWidget/Resource/TimeStatisticOverTimes.png&quot;);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_Overtimes">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>13</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:13pt;</string>
                </property>
                <property name="text">
                 <string> 0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Agency FB</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:10pt;
color:#999999;</string>
                </property>
                <property name="text">
                 <string>Timeout</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLogoButton</class>
   <extends>QWidget</extends>
   <header>UI/MyWidget/MyLogoButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
